server/controllers/finance/reports/reportAccounts/index.js

```diff
@@ -67,54 +62,49 @@ function document(req, res, next) {
 /**
  * @function queryReportAccount
  * This feature select all transactions for a specific account
-*/ 
-function queryReportAccount(accountId, source){
+*/
+function queryReportAccount(accountId, source) {
   source = parseInt(source);
+  let sql;
 
-  if(source === 1){
-
-    let sql = `
+  if (source === 1) {
```

Comment:
In my view, I would do that as a separate PR.  I have no idea if these queries even produce correct data - a separate PR could do a full audit.